<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clsRunningStatus</class>
 <widget class="QDialog" name="clsRunningStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>166</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>166</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/res.qrc">
    <normaloff>:/lightWave.png</normaloff>:/lightWave.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-color:rgb(150,150,251);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="labelChannel">
       <property name="minimumSize">
        <size>
         <width>88</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;楷体&quot;;
background-color: rgb(255, 255, 0);</string>
       </property>
       <property name="text">
        <string>通道 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;楷体&quot;;</string>
       </property>
       <property name="title">
        <string>测试结果</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <property name="spacing">
         <number>3</number>
        </property>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="labelItem1">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>56</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>L:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelRes1">
            <property name="minimumSize">
             <size>
              <width>108</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>108</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>100.88uH</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelStatus1">
            <property name="maximumSize">
             <size>
              <width>56</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>PASS</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="labelItem2">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Q:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelRes2">
            <property name="minimumSize">
             <size>
              <width>108</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>108</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>36.6Q</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelStatus2">
            <property name="maximumSize">
             <size>
              <width>56</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>FAIL</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images/res.qrc"/>
  <include location="../images/res.qrc"/>
  <include location="../images/res.qrc"/>
 </resources>
 <connections/>
</ui>
